tools/detect_nonexistent_modules_in_yaml_schedule

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env perl
+
+# The script verifies if modules used in yaml schedule do not exist in the repo.
+
+use strict;
+use warnings;
+use YAML::Tiny;
+use File::Basename;
+use Data::Dumper;
+
+=head2 parse_modules_in_schedule
+
+   parse_modules_in_schedule($schedule_file_path);
+
+Get test modules list from 'schedule' and 'conditional_schedule' sections of
+YAML schedule file.
+
+=cut
+sub parse_modules_in_schedule {
+    my ($schedule_file_path) = @_;
+    my $schedule = YAML::Tiny::LoadFile($schedule_file_path);
+    my @scheduled;
+    for my $module (@{$schedule->{schedule}}) {
+        push(@scheduled, $module) && next unless ($module =~ s/\{\{(.*)\}\}/$1/);
+        # Module is scheduled conditionally. Need to be parsed. Get condition hash
+        my $condition = $schedule->{conditional_schedule}->{$module};
+        # Iterate over variables in the condition
+        foreach my $var (keys %{$condition}) {
+            foreach my $val (keys %{$condition->{$var}}){
+                # Iterate over the list of the modules to be loaded
+                push(@scheduled, $_) for (@{$condition->{$var}->{$val}});
+            }
+        }
+    }
+    return @scheduled;
+}
+
+=head2 is_test_module_exist
+
+   is_test_module_exist($test_module);
+
+Returns true if test module with C<$relative_file_path> path exists in project
+C<$test_module> is file path relative to 'tests' directory, without file
+extension (as specified in YAML schedule file, e.g. 'boot/boot_to_desktop').
+
+=cut
+sub is_test_module_exist {
```

Comment:
s/is_test_module_exit/exists_test_module/ or something like that.

Reply:
I'm not against your suggestion, I just like convention when all the boolean methods start with is_, has_.

Reply:
I know that :), I used to use in other languages `is_present`, so it would be something like `is_present_module` but leave it as it is i you still like more your version.